queenbee/recipe/dag.py

```diff
@@ -381,31 +22,12 @@ class DAGTaskLoop(BaseModel):
     input or task parameter which should be a list.
     """
 
-    from_: Union[InputParameterReference, TaskParameterReference] = Field(
+    from_: Union[InputReference, TaskReference, ValueListReference] = Field(
```

Comment:
Am I right in thinking that the three references types all represent `parameters` and not `artifacts`?

Reply:
Yes. If you don't see Folder/File/Path in the name of the reference then it is referring to a parameter.